core/src/apps/common/definitions_constants.py

```diff
@@ -9,7 +9,7 @@
     b"\xb8\xd2\xb2\x1d\xe2\x71\x24\xf0\x51\x1f\x90\x3a\xe7\xe6\x0e\x07\x96\x18\x10\xa0\xb8\xf2\x8e\xa7\x55\xfa\x50\x36\x7a\x8a\x2b\x8b",
 )
 
-MIN_DATA_VERSION = 1738953548
+MIN_DATA_VERSION = 1725628031
```

Comment:
IIUC, the above diff seems to be unrelated to this PR...

I ran `make templates` and this file got updated back to 1738953548
(which matches `2025-02-07T18:39:08+00:00` from `common/defs/ethereum/released-definitions-timestamp.txt`)

Reply:
removed in rebase